HerPublicWebsite/Views/ReferralRequestFollowUp/ResponsePage.cshtml

```diff
@@ -11,10 +11,10 @@
 
 <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-        <p class="govuk-body-m">@Model.ReferralCode</p>
+        <p class="govuk-body-m">Reference code: <strong class="govuk-!-font-weight-bold">@Model.ReferralCode</strong></p>
```

Comment:
I think it would be more standard to have the "Reference code:" be in bold, since that is the label.

Reply:
Ah, that seems reasonable. I'd done it this way to match the style in the Confirmation page. But the setup and context of that is a little different so it might not make sense to do it here.

<img width="972" alt="image" src="https://github.com/UKGovernmentBEIS/desnz-home-energy-retrofit-beta/assets/104021096/8f595530-ef5c-4cdd-9e6a-56d66028d72c">

Reply:
Have committed the new style. Will update the Jira screenshot as well.